persistence/jpa/src/main/java/org/infinispan/persistence/jpa/configuration/JpaStoreConfigurationSerializer.java

```diff
@@ -0,0 +1,32 @@
+package org.infinispan.persistence.jpa.configuration;
+
+import javax.xml.stream.XMLStreamException;
+
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.configuration.serializing.AbstractStoreSerializer;
+import org.infinispan.configuration.serializing.ConfigurationSerializer;
+import org.infinispan.configuration.serializing.XMLExtendedStreamWriter;
+
+/**
+ * JpaStoreConfigurationSerializer.
+ *
+ * @since 9.0
+ * @author Tristan Tarrant
+ */
+public class JpaStoreConfigurationSerializer extends AbstractStoreSerializer implements ConfigurationSerializer<JpaStoreConfiguration> {
+
+   @Override
+   public void serialize(XMLExtendedStreamWriter writer, JpaStoreConfiguration configuration) throws XMLStreamException {
+      AttributeSet attributes = configuration.attributes();
+      writer.writeStartElement(Element.JPA_STORE);
+      attributes.write(writer, JpaStoreConfiguration.ENTITY_CLASS, Attribute.ENTITY_CLASS_NAME);
```

Comment:
you don't need to write the attributes individually. just do ```configuration.attributes().write(writer)``` otherwise the abstract store attributes are not serialized.